0001-basic-expiry-model.md

```diff
@@ -0,0 +1,160 @@
+
+#### Model
+
+To determine `beta_expiry`, the random variable <img src="https://latex.codecogs.com/gif.latex?R_{\beta}"/>, which models the time Alice will need to get her desired number of confirmations on her redeem transaction of `beta_asset`, will need to be approximated.
+
+The time it would take for Alice's redeem transaction to obtain <img src="https://latex.codecogs.com/gif.latex?k"/> confirmations on `beta_ledger` can be modelled as a sum of <img src="https://latex.codecogs.com/gif.latex?k"/> mutually independent exponential random variables with the same rate parameter <img src="https://latex.codecogs.com/gif.latex?\lamda"/>, which is equal to an <img src="https://latex.codecogs.com/gif.latex?\textrm{Erlang}(k,\lambda)"/> distribution.
+
+Letting <img src="https://latex.codecogs.com/gif.latex?C_{k}\sim\textrm{Erlang}(k,\lambda)"/>,
+
+<img src="https://latex.codecogs.com/gif.latex?E_{\beta}=t_{0}&plus;T^{\alpha}_{A}&plus;T^{\beta}_{B}&plus;T^{\beta}_{A}&plus;Q_{C_{k}}(p)"/>, [1]
+
+where <img src="https://latex.codecogs.com/gif.latex?t_{0}"/> is the time when all the protocol parameters are set and <img src="https://latex.codecogs.com/gif.latex?Q_{C_{k}}(p)"/> is the [quantile function](https://en.wikipedia.org/wiki/Quantile_function) of <img src="https://latex.codecogs.com/gif.latex?C_{k}"/> for a probability <img src="https://latex.codecogs.com/gif.latex?p"/>.
+This is, a function returning the minimum time taken for <img src="https://latex.codecogs.com/gif.latex?C_{k}"/> to take place with probability <img src="https://latex.codecogs.com/gif.latex?p"/>.
+
+As mentioned in the previous section, quantile functions for Erlang random variables can only be approximated via [numerical methods](https://docs.scipy.org/doc/scipy-0.16.1/reference/generated/scipy.stats.erlang.html).
+
+
+With this knowledge, `beta_expiry` can be estimated using [1].
+
+### Example
+
+All times expressed in minutes.
+
+#### Code
+
+```python
+from scipy.stats import erlang
+
+# beta expiry
+# E_{\beta} = t_{0} + T^{\alpha}_{A} + T^{\beta}_{B} + T^{\beta}_{A} + Q_{C_{k}}(p)
+
+# Time when all parameters will be set and the protocol will commence
+t_0 = 0
+# Time in which Alice is overwhelmingly confident she will be able to confirm a Bitcoin (alpha ledger) transaction
+alice_alpha_time = 20
+# Time in which Bob is overwhelmingly confident he will be able to confirm an Ethereum (beta ledger) transaction
+bob_beta_time = 2
+# Time in which Alice is overwhelmingly confident she will be able to confirm an Ethereum (beta ledger) transaction
+alice_beta_time = 1.5
+# Number of additional confirmations Alice needs on her redeem transaction of beta_asset
+alice_confirmations = 40
+# Ethereum (beta ledger) block time
+beta_block_time = 0.25
+# Time taken for k blocks to be generated on Ethereum (beta ledger) with probability 0.95
+time_for_confirmations_beta = erlang.ppf(0.999999, alice_confirmations, 0, beta_block_time)
```

Comment:
🔧 
1. This comment is stale :)
2. extract the probability into a variable
3. We want Bob to lose money with much smaller probability than the trade being canceled. (e.g. 1/1_000 vs 1_000_000).

Reply:
Very good points!